src/main/java/subway/stations/ClosestStations.java

```diff
@@ -0,0 +1,49 @@
+package subway.stations;
+
+import subway.stations.SubwayStations.Station;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ClosestStations {
+
+    List<Double> currentCoordinates;
+    List<Double> destinationCoordinates;
+
+    public List<Station> getClosestStations(List<Double> currentCoordinates, List<Double> destinationCoordinates, SubwayStations subwayStations){
```

Comment:
It would be better (and simpler) to have one method `getClosestStation(subwayStations, coordinates)` and just call it twice.